akka-testkit-typed/src/main/scala/akka/testkit/typed/javadsl/BehaviorTestKit.scala

```diff
@@ -63,6 +63,12 @@ abstract class BehaviorTestKit[T] {
    */
   def expectEffect(expectedEffect: Effect): Unit
 
+  /**
+   * Asserts that the oldest effect is the of class T. Consumes and returns the concrete effect for
```

Comment:
oldest? would "first" be better wording?

"the of" ?

Reply:
ups copy pasta 🍝